File: backend/src/main/java/com/ssafy/zip/entity/Comment.java
package com.ssafy.zip.entity;

import lombok.*;

import javax.persistence.*;
import java.time.LocalDateTime;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Table(name = "comment")
public class Comment {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long id;

    Long boardId;

    @ManyToOne
    @JoinColumn(name = "user_id")
    User user;

    String content;

    LocalDateTime reg;
}

File: android/app/src/main/java/com/ssafy/zip/android/data/Comment.kt
package com.ssafy.zip.android.data

import android.os.Parcelable
import kotlinx.parcelize.Parcelize
import java.util.Date

@Parcelize
data class Comment(
    val content : String,
    val id : Long,
    val reg : Date,
    val user : User,
) : Parcelable
